app/views/admin/communities/edit_welcome_email.haml

```diff
@@ -34,7 +34,7 @@
   - content_for(:resend_link) do
     = link_to(t("admin.communities.outgoing_email.resend_link"), nil, class: "js-sender-address-resend-verification")
 
-  - if user_defined_address.nil?
+  - if user_defined_address.nil? || !ses_in_use
```

Comment:
I'm guessing that if user defined address is not set we show controls to set it. If it's set, then we don't show the controls because chancing address set one time is not supported by the ses integration at the moment. This needs manual intervention from us.

Should this be:
```ruby
if user_defined_address.nil? && ses_in_use
```

So if user hasn't defined an address and we have ses in use then show the controls? Or maybe I'm misinterpreting this but it's hard to tell what shown and hidden just from the haml below.